cfgov/jinja2/v1/_includes/organisms/header.html

```diff
@@ -1,5 +1,41 @@
 <header class="o-header"
         role="banner">
+
+    {% if flag_enabled('BETA_NOTICE') %}
```

Comment:
Cool! We'll have to think about how to handle optional HTML on the JS-side. Like if `if flag_enabled('BETA_NOTICE')` is false, the HTML won't render and the JS behaviors attached to the HTML will error and stop all further scripts on the page from loading, essentially breaking the page. I've handled this in a `try…catch` block in this PR, but that should ideally be handled on the backend to exclude the needed JS when `BETA_NOTICE` is absent.